GameDev/week3/livestream/script.md

```diff
@@ -1 +1,266 @@
-Put your script here.
+# Intro
+
+- *Introduce yourselves*
```

Comment:
Awesome formatting for the text, but same issue with code formatting here. Add syntax highlighting: https://www.markdownguide.org/extended-syntax/#syntax-highlighting